### DISCUSSION TITLE
Tab Panels - using v-show instead of v-if

### DISCUSSION THREAD

**Discussion by deepreshion** [quasarframework/quasar#16721] [General - Components / Directives / etc]
The tabs for switching are using v-if, which is causing part of my functionality to not work correctly (keep-alive won't help here). Is there a possibility to configure this logic, or is the only option to create custom tabs instead of using the ready-made?

**Comment by ghiscoding** [comment]
duplicate of #11252 and I would also like to know how to achieve this, but I didn't find anything so far :(
